index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Product Landing Page</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      Explore
+      Start a project
+      <img src="logo.png">
+      <input type="text" placeholder="Search">
+      Log in
+    </nav>
+    <h1>The Minimalist’s Wall Calendar | Visualize Your Year</h1>
+    <p>Minimalist, forgiving, and beautiful. A wall calendar designed to make 2021 feel like a breath of fresh air.</p>
+  </header>
+  <main>
+    <figure>
+      <img src="hero.png">
+      <figcaption>Project we Love | Product Design | Cambridge, MA</figcaption>
+    </figure>
+    <aside>
+      <hr>
+      <p>
+        $13,757<br>
+        pledged of $1,200 goal
+      </p>
+      <p>
+        215 <br>
+        backers
+      </p>
+      <p>
+        19 <br>
+        days to go
+      </p>
+      <input type="button" value="Back this Project"><br>
+      <input type="button" value="Remind me">
+      <p>
+        <sub>All or nothing. This project will only be funded if it reaches its goal by Thu, November 26 2020 1:35 PM
+          CST.</sub>
+      </p>
+    </aside>
+    <nav>
+      Campaign | FAQ | Updates<sup>2</sup> | Comments<sup>19</sup> | Community
+    </nav>
+    <aside>
+      <nav>
+        <p>STORY</p>
+        <hr>
+        <p>RISKS</p>
+        <hr>
+        <p>ENVIRONMENTAL<br>
+          COMMITMENTS</p>
+      </nav>
+    </aside>
+    <article>
+      <p>Story</p>
```

Comment:
In keeping with my comments making you consider meaning. 

Is this text the same level of importance as the rest of the `p` tags in this article? Or is it more important? If it's more important, you should make it a heading to indicate that it's an important piece of information related to the rest of the content.

I'd recommend reviewing the `heading` article on MDN to check on which heading level it should be, and also to potentially think about how you organize the page.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements